presto-hive-metastore/src/main/java/com/facebook/presto/hive/MetastoreClientConfig.java

```diff
@@ -54,6 +55,12 @@ public class MetastoreClientConfig
     private boolean deleteFilesOnTableDrop;
     private boolean invalidateMetastoreCacheProcedureEnabled;
 
+    private boolean tlsEnabled;
+    private File keystorePath;
+    private String keystorePassword;
+    private File truststorePath;
+    private String trustStorePassword;
+
```

Comment:
This comment is for line 64-74
Move it to the new com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastoreConfig.java